common/perception_utils/include/perception_utils/perception_utils.hpp

```diff
@@ -102,6 +104,72 @@ inline double get2dIoU(const T1 source_object, const T2 target_object)
   return iou;
 }
 
+inline std::map<std::string, double> getMaxMin2d(const tier4_autoware_utils::Polygon2d & polygon)
+{
+  const auto init_point = polygon.outer().at(0);
+  double max_x = init_point.x();
+  double min_x = init_point.x();
+  double max_y = init_point.y();
+  double min_y = init_point.y();
+  for (const auto & point : polygon.outer()) {
+    if (point.x() > max_x) max_x = point.x();
+    if (min_x > point.x()) min_x = point.x();
+    if (point.y() > max_y) max_y = point.y();
+    if (min_y > point.y()) min_y = point.y();
+  }
+  std::map<std::string, double> max_min_map;
+
+  max_min_map.insert(std::make_pair("max_x", max_x));
+  max_min_map.insert(std::make_pair("min_x", min_x));
+  max_min_map.insert(std::make_pair("max_y", max_y));
+  max_min_map.insert(std::make_pair("min_y", min_y));
```

Comment:
I don't think it's a good idea to have max/min_x/y as a map. (In addition, maybe better to use unordered_map than map)

How about creating a struct, instead of map like
```
struct MinMaxPosition
{
  double max_x;
  double min_x;
  double max_y;
  double min_y;
};
```

Reply:
reflect the indication in https://github.com/autowarefoundation/autoware.universe/pull/1902/commits/975e98db152e6229c8f8f3cb674e93f98ff007ee